src/main/java/com/google/sps/servlets/MenteeSignupServlet.java

```diff
@@ -32,6 +33,45 @@
 @WebServlet("/mentee-signup")
 public class MenteeSignupServlet extends HttpServlet {
 
+  /**
+   * Gets majors from database and returns as JSON.
+   */
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    // Set up map to store majors.
+    Map<Integer, String> majors = new HashMap<Integer, String>();
+    
+    // Set up variables needed to connect to MySQL database.
+    String url = Utility.SQL_LOCAL_URL;
+    String user = Utility.SQL_LOCAL_USER;
+    String password = Utility.SQL_LOCAL_PASSWORD;
```

Comment:
Just like in previous PRs, no need to reallocate these variables locally - you can just directly use the constants defined in the Utility class.

Reply:
Done!